src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -95,6 +96,10 @@ class IOUAmountPage extends React.Component {
         this.focusTextInput();
     }
 
+    get formattedAmount() {
+        return this.replaceAllDigits(this.state.amount, this.props.toLocaleDigit);
+    }
```

Comment:
We haven't used getters till now. So for consistency, we can convert this to a normal class variable and move it in render.